runtime-config-linux.md

```diff
@@ -131,6 +131,58 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+The path to the cgroups can to be specified in the Spec via `cgroupsPath`.
+`cgroupsPath` is expected to be relative to the cgroups mount point.
+If not specified, cgroups will be created under '/' with the base name of the bundle.
```

Comment:
On Thu, Sep 10, 2015 at 01:54:07AM -0700, Daniel, Dao Quang Minh wrote:
> > +If not specified, cgroups will be created under '/' with the base name of the bundle.
> 
> I think it's more reasonable / simpler to error out here rather than
> continuing and generate a path based on base name of the
> bundle. `cgroupsPath` should be a required data right ?

I often want cgroups for my containers, but don't care what the path
is.  I think we do want the ability to tell the runtime “I don't care,
just make something up”, which is what leaving cgroupsPath out does.
But I'd rather leave the “make something up” algorithm up to the
runtime [1].  See also the “several containers from one bundle” use
case [2].

[1]: https://github.com/opencontainers/specs/pull/137#issuecomment-139103731
[2]: https://github.com/opencontainers/specs/pull/137#issuecomment-139078372